src/pages/plugins/index.js

```diff
@@ -95,21 +101,28 @@ function PluginFooter({ keywords = [] }) {
 }
 
 function PluginComponent({ plugin }) {
+  const deprecatedCard = plugin.deprecatedMessage ? true : false;
+  const deprecatedMessage = deprecatedCard && plugin.deprecatedMessage;
+  const cardClasses = classnames('card', { [styles.pluginDeprecatedCard]: plugin.deprecatedMessage ? true : false });
+  const cardHeader = classnames( 'card__header', deprecatedCard ? styles.deprecatedCardHeader : styles.pluginCardHeader );
+  const cardBody = classnames( deprecatedCard ? styles.pluginCardFlagMessage : 'card__body' );
+
   return (
     <div className="row padding-vert--md">
       <div className="col col--10 col--offset-1">
-        <div key={plugin.key} className="card">
-          <div className={classnames('card__header', styles.pluginCardHeader)}>
+        <div key={plugin.key} className={cardClasses}>
+          <div className={cardHeader}>
+            {deprecatedCard && <span className="badge badge--primary">Deprecated</span>}
             <h2><a className={styles.primaryUrl} href={plugin.primaryUrl}>{plugin.name}</a></h2>
-            <span className="badge badge--primary">{plugin.version}</span>
+            {deprecatedCard || <span className="badge badge--primary">{plugin.version}</span>}
```

Comment:
I don't think `||` is easy to understand here. Maybe you can use `!isDeprecated && <span className="badge badge--primary">{plugin.version}</span>` instead?

Reply:
The reason I write || is to trigger following component if deprecatedCard is false. Which || means or so if A || B is A or B so if A is false it will check the if B is true. Also it worked but want to ask if what you worrying about is the visibility? I implemented with ! and && and just want to know about the reason.

Reply:
The only reason I suggested was for other people that work on your code. It might be more helpful for them to understand.